steps/aws-ec2-step-instances-describe/step.yaml

```diff
@@ -57,4 +57,40 @@ examples:
     spec:
       aws:
         connection: !Connection { type: aws, name: my-aws-account }
-        region: !Parameter awsRegion
\ No newline at end of file
+        region: !Parameter awsRegion
+- summary: List running EC2 instances
+  content:
+    apiVersion: v1
+    kind: Step
+    name: describe-instances
+    image: relaysh/aws-ec2-step-instances-describe
+    spec:
+      aws:
+        connection: !Connection { type: aws, name: my-aws-account }
+        region: !Parameter awsRegion
+      filters:
+        instance-state-name: "running"
+- summary: List EC2 instances created by a specified keypair
+  content:
+    apiVersion: v1
+    kind: Step
+    name: describe-instances
+    image: relaysh/aws-ec2-step-instances-describe
+    spec:
+      aws:
+        connection: !Connection { type: aws, name: my-aws-account }
+        region: !Parameter awsRegion
+      filters:
+        key-name: !Parameter keyName

```

Comment:
Our keys should be camelCase instead of hyphen case. So this would be keyName.

Reply:
So, I'm passing these transparently from AWS -> API call (e.g. key-name is how AWS defines the filter). I could create a mapping from camelCase to hyphen case, but just wondered if that was more confusing for users.